sdk/redisenterprise/Azure.ResourceManager.RedisEnterprise/tests/ScenarioTests/AuthenticationFuntionalTests.cs

```diff
@@ -0,0 +1,172 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading.Tasks;
+using Azure.Core.TestFramework;
+using Azure.ResourceManager.RedisEnterprise.Models;
+using Azure.ResourceManager.Resources;
+using NUnit.Framework;
+
+namespace Azure.ResourceManager.RedisEnterprise.Tests.ScenarioTests
+{
+    public class AuthenticationFuntionalTests : RedisEnterpriseManagementTestBase
+    {
+        public AuthenticationFuntionalTests(bool isAsync)
+                   : base(isAsync, RecordedTestMode.Record)
+        {
+        }
+
+        private ResourceGroupResource ResourceGroup { get; set; }
+
+        private RedisEnterpriseClusterCollection Collection { get; set; }
+
+        private async Task SetCollectionsAsync()
+        {
+            ResourceGroup = await CreateResourceGroupAsync();
+            Collection = ResourceGroup.GetRedisEnterpriseClusters();
+        }
+
+        [Test]
+        public async Task AuthenticationTests()
+        {
+            await SetCollectionsAsync();
```

Comment:
Please follow https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/resourcemanager/Azure.ResourceManager/docs/TestGuide.md to record the test

Reply:
done